tools/SetupFlow/DevHome.SetupFlow/Utilities/DevDriveUtil.cs

```diff
@@ -6,11 +6,11 @@
 using System.IO;
 using System.Linq;
 using System.Runtime.InteropServices;
-using CommunityToolkit.Common;
 using Serilog;
 using Windows.Win32;
 using Windows.Win32.Foundation;
 using Windows.Win32.UI.Shell;
+using static CommunityToolkit.Common.Converters;

```

Comment:
Curious: Why switch to static? Advantage?

Reply:
This one is because I created a new folder under `tools/SetupFlow/DevHome.SetupFlow/` called "Converters". So that caused a namespace conflict with the `CommunityToolkit.Common.Converters` namespace in this file. I moved it to static since we only use one static method from it in this file. It was either that or do  `CommunityToolkit.Common.Converters.<methodName>` I just chose the former.